LICENSE.txt

```diff
@@ -4,11 +4,11 @@
                            Version 2.0, January 2004
                         http://www.apache.org/licenses/
 
-   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```

Comment:
It was not meant to make any changes in the license . Please ommit it.

Reply:
You have to undo the changes by yourself, we cannot just ignore this as the squash merge will treat everything as a block.

Reply:
I just removed the changes, will it work ??. as I have edited the many files altogether and I dont know how to revert changes for a single file. If possible please help.

Reply:
There seems to be other changes affecting this file, remove them too and we will be good to go :)